src/variables.json

```diff
@@ -60,9 +53,9 @@
   "white": "#fff",
   "transparent": "transparent",
 
-  "neutral10": "rgba(127, 127, 127, 0.1)",
-  "neutral25": "rgba(127, 127, 127, 0.25)",
-  "neutral45": "rgba(127, 127, 127, 0.45)",
+  "disabled-primary-interactive-color": "rgba(127, 127, 127, 0.25)",
+  "disabled-secondary-interactive-color": "rgba(127, 127, 127, 0.1)",
+  "disabled-text-color": "rgba(127, 127, 127, 0.45)",
```

Comment:
Is `disabled` accurate? This is used for placeholder too right? And I'm not sure how `primary` & `secondary` align with how they are used. How about `input-{value}-on-blur`?